UKHO.ExchangeSetService.API/UKHO.ExchangeSetService.Common.UnitTests/Helpers/FileShareServiceTests.cs

```diff
@@ -1150,6 +1159,107 @@ public async Task WhenGetBatchInfoBasedOnProductsWithCancellation_ThenReturnsSea
             Assert.AreEqual("13d38bde-5191-4a59-82d5-aa22ca1cc6de", response.Entries[1].BatchId);
         }
 
+        [Test]
+        public async Task WhenValidUploadLargeMediaZipFileRequest_ThenReturnTrue()
+        {
+            fakeFileShareConfig.Value.BlockSizeInMultipleOfKBs = 256;
+            fakeFileShareConfig.Value.ParallelUploadThreadCount = 0;
+            byte[] byteData = new byte[1024];
+
+            var responseBatchStatusModel = GetBatchStatusResponse();
+            var jsonString = JsonConvert.SerializeObject(responseBatchStatusModel);
+            var httpResponse = new HttpResponseMessage() { StatusCode = HttpStatusCode.Created, Content = new StreamContent(new MemoryStream(Encoding.UTF8.GetBytes(jsonString))) };
```

Comment:
are we not checking that a 201 has been given

Reply:
We are not checking because that httpResponse is used to return fake response for http call.